backend/src/main/java/com/kodstar/backend/model/entity/IssueEntity.java

```diff
@@ -1,4 +1,43 @@
 package com.kodstar.backend.model.entity;
 
+
+import com.kodstar.backend.model.enums.Label;
+
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "issue")
 public class IssueEntity {
+
+    @Id
+    @NotNull
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotBlank
+    private String tittle;
```

Comment:
suggestion
--> should be written like this "title"

Reply:
will be fixed the typo. Thanks